app/Http/Controllers/API/ConferencesController.php

```diff
@@ -7,24 +7,17 @@
 use Carbon\Carbon;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
+use App\Http\Requests\StoreConferenceRequest;
 
 class ConferencesController extends Controller
 {
-    public function store(Request $request)
+    public function store(StoreConferenceRequest $request)
     {
-        $after = (new Carbon($request->input('start_date')))->subDay();
```

Comment:
I typically don't care all that much if there is a simple validation block in the controller. Usually about 3 rules is my break unless there is something a bit more complicated that I don't want to have to concern myself with in the controller. In this case, it was the temp variable `$after` that prompted me to move the validation out to it's own Form Request. 

As far as discovering the validation rules... eh... I guess if you used Form Requests anywhere in the project and weren't aware of their presence you would have the same issue. Judgement call. 

Let me know what you want done and I'll make it happen 😄👍